docker-compose.yml

```diff
@@ -5,7 +5,7 @@ volumes:
 
 services:
   db:
-    image: postgres:13.5-alpine
+    image: postgres:13.18-alpine
```

Comment:
N.B. In the dockerfile we're already using the relevant postgres driver for this